### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by DivyanshSareen**

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -169,4 +169,25 @@
...
     background: var(--grey);
-}
\ No newline at end of file
+}
+
+/*-- Ghost Button /'''\(Ⓘ.Ⓘ)/'''\ spooky--*/
+
+.btn-ghost{
+    color: var(--primary-900);
+    filter: none;
```
Comment (sanjitsarkar) [inline components/button.css:178]: filter:none;
Doesn't have any effect.

**Review by sanjitsarkar — CHANGES_REQUESTED** [review]

File: components/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+/* SNACK-BAR (・ω・)⊃- [二> >二] -⊂(´∀｀)*/
+
+/* styling and positioning snackbar */
+.snack{
+    background-color: #2b2b2b;
+    margin: 0 4em 2em 4em;
+    padding: 0 1em 0 1em ;
+    border-radius: var(--radius-s);
+
```
Comment (sanu29) [inline components/snackbar.css]: extra spaces

**Review by sanu29 — APPROVED** [review]
Looks good for me

File: components/button.css (hunk 1 above)
Comment (DivyanshSareen) [reply components/button.css:178]: it removes drop shadow from button component

Comment (sanjitsarkar) [reply components/button.css:178]: I think You can also use border:none;
It also does the job.

Comment (DivyanshSareen) [reply components/button.css:178]: I tried border none, it is not working.

**Review by sanjitsarkar — APPROVED** [review]

**Pull request merged**